var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -103,8 +103,8 @@ class Lbann(CMakePackage, CudaPackage, ROCmPackage):
     conflicts('+gold', when='platform=darwin', msg="gold does not work on Darwin")
     conflicts('+lld', when='platform=darwin', msg="lld does not work on Darwin")
 
-    depends_on('cmake@3.21.0: ~doc', type='build', when='@0.103:')
-    depends_on('cmake@3.17.0: ~doc', type='build', when='@:0.102')
+    depends_on('cmake@3.21.0:', type='build', when='@0.103:')
+    depends_on('cmake@3.17.0:', type='build', when='@:0.102')
```

Comment:
While we are at it we can simplify like:
```python
depends_on('cmake@3.17.0:', type='build')
depends_on('cmake@3.21.0:', type='build', when='@0.103:')
```
i.e. the constraint on `cmake` tightens with increasing versions of `lbann`.

Reply:
@alalazo Done.